### PR TITLE
fix: sonar issue reorder

### PR DESCRIPTION
**Pull request by Angel-M-R**
Fix issue #180

### CODE REVIEW

File: src/main/java/org/trebol/integration/mailing/mailgun/MailgunMailingServiceImpl.java [hunk 1]
```diff
@@ -59,6 +59,9 @@
 public class MailgunMailingServiceImpl
   implements IMailingIntegrationService {
 
+  private final static String OWNERS_MAPS_KEY_PREFIX = "owners:";
+  private final static String CUSTOMER_MAPS_KEY_PREFIX = "customer:";
```
Comment (mepox) [inline src/main/java/org/trebol/integration/mailing/mailgun/MailgunMailingServiceImpl.java:63]: The modifiers needs to be in the correct order. The modifier "static" comes before the "final".

Source: https://sonarcloud.io/organizations/trebol-ecommerce/rules?open=java%3AS1124&rule_key=java%3AS1124

**Review by mepox — CHANGES_REQUESTED** [review]
Hi @Angel-M-R ! Please see my comment.

File: src/main/java/org/trebol/integration/mailing/mailgun/MailgunMailingServiceImpl.java (hunk 1 above)
Comment (Angel-M-R) [reply src/main/java/org/trebol/integration/mailing/mailgun/MailgunMailingServiceImpl.java]: Done, thank you

**Review by bglamadrid — APPROVED** [review]
This looks good now. Thanks @mepox and @Angel-M-R!

**Pull request merged**